nautobot_device_lifecycle_mgmt/tables.py

```diff
@@ -220,12 +220,31 @@ class Meta(BaseTable.Meta):  # pylint: disable=too-few-public-methods
 class DeviceSoftwareValidationResultTable(BaseTable):
     """Table for device software validation report."""
 
-    name = tables.Column(accessor="device__device_type__model", verbose_name="Device Type")
-    total = tables.Column(accessor="total", verbose_name="Total")
-    valid = tables.Column(accessor="valid", verbose_name="Valid")
-    invalid = tables.Column(accessor="invalid", verbose_name="Invalid")
-    no_software = tables.Column(accessor="no_software", verbose_name="No Software")
+    name = tables.TemplateColumn(
+        template_code='<a href="/dcim/devices/?device_type_id={{ record.device__device_type__pk }}">{{ record.device__device_type__model }}</a>'
+    )
+    total = tables.TemplateColumn(
+        template_code='<a href="/plugins/nautobot-device-lifecycle-mgmt/device-validated-software-result/'
+        '?&device_type={{ record.device__device_type__model }}">{{ record.total }}</a>'
+    )
```

Comment:
Should we use `record.device__device_type__pk` for url?

Reply:
When I switch it to that I get a filter error.

![image](https://github.com/nautobot/nautobot-plugin-device-lifecycle-mgmt/assets/76115401/8b6c5709-f210-460d-b64e-5d0ed9a7eb49)

Reply:
let's keep this as model for now and can review later if it makes sense to change.